turbinia/turbiniactl.py

```diff
@@ -72,6 +73,10 @@ def main():
       'will ignore config files in other default locations '
       '(/etc/turbinia.conf, ~/.turbiniarc, or in paths referenced in '
       'environment variable TURBINIA_CONFIG_PATH)', required=False)
+  parser.add_argument(
```

Comment:
Agree. I assume keeping the recipe_config parameter makes sense, because you can have a client pass an ad-hoc dict. If so, I have included a check to make sure --recipe and --recipe_config are not specified on the same submission, along with the checks you suggested.